app/src/main/res/layout-v17/tool_detail_banner_video_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView
+        android:id="@+id/video_banner"
+        android:adjustViewBounds="true"
+        android:background="@color/gray_E6"
+        android:scaleType="centerCrop"
+        app:autoPlay="false"
+        app:layout_aspectRatio="210%"
+        app:layout_widthPercent="100%"
+        app:showFullScreenButton="false" />
+
+</merge>
```

Comment:
in this specific layout you don't need the `<merge>` tags, this can just be the `YouTubePlayerView`